support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx

```diff
@@ -18,7 +18,7 @@ import { gaEvent } from 'helpers/tracking/googleTagManager';
 import * as storage from 'helpers/storage';
 import { set as setCookie } from 'helpers/cookie';
 import Page from 'components/page/page';
-import Footer from 'components/footer/footer';
```

Comment:
is this component still used elsewhere?

Reply:
looks like just the paypal error page, and the TY page